concepticondata/conceptlists/Nagano-2013-1256.tsv

```diff
@@ -0,0 +1,1257 @@
+Nagano-2013-1256-206	206	0376	eat (vt)	1336	EAT	
+Nagano-2013-1256-207	207	0378	lick (vt)	319	LICK	
+Nagano-2013-1256-208	208	0380	chew (vt)	321	CHEW	
+Nagano-2013-1256-209	209	0381	masticate (vt)			
+Nagano-2013-1256-210	210	0383	drink (vt)	1401	DRINK	
+Nagano-2013-1256-211	211	0384	swallow (vt)	897	SWALLOW	
+Nagano-2013-1256-212	212	0386	suck (vt)	1421	SUCK	
+Nagano-2013-1256-213	213	0388	vomit (vt)/vomit (呕吐) (vt)	1278	VOMIT	
+Nagano-2013-1256-214	214	0390	spit (v)	1440	SPIT	
+Nagano-2013-1256-215	215	0392	get hungry (v)			
+Nagano-2013-1256-216	216	0393	feel hungry (v)	1533	BE HUNGRY	
+Nagano-2013-1256-217	217	0395	feel thirsty (v)	847	THIRSTY	
+Nagano-2013-1256-218	218	0397	taste	2190	TASTE	
+Nagano-2013-1256-219	219	0399	tasty	1813	TASTY	
+Nagano-2013-1256-220	220	0401	sweet	717	SWEET	
+Nagano-2013-1256-221	221	0403	hot	1286	HOT	
+Nagano-2013-1256-222	222	0405	bitter	887	BITTER	
+Nagano-2013-1256-223	223	0407	sour	1906	SOUR	
+Nagano-2013-1256-224	224	0409	rotten	1728	ROTTEN	
+Nagano-2013-1256-225	225	0411	mould	1166	MOULD	
+Nagano-2013-1256-226	226	0414	house	1252	HOUSE	
+Nagano-2013-1256-227	227	0415	room	1450	ROOM	
+Nagano-2013-1256-228	228	0417	house, residence	2406	HOME	
+Nagano-2013-1256-229	229	0419	residence			
+Nagano-2013-1256-230	230	0420	community, settlement			
+Nagano-2013-1256-231	231	0423	tent	1316	TENT	
+Nagano-2013-1256-232	232	0425	build (a house) (vt)			
+Nagano-2013-1256-233	233	0427	entrance			
+Nagano-2013-1256-234	234	0428	exit			
+Nagano-2013-1256-235	235	0430	door	1567	DOOR	
+Nagano-2013-1256-236	236	0432	room	1450	ROOM	
+Nagano-2013-1256-237	237	0434	floor	1632	FLOOR	
+Nagano-2013-1256-238	238	0436	pillar			
+Nagano-2013-1256-239	239	0437	beam (of a house)	1132	BEAM	
+Nagano-2013-1256-240	240	0440	wall	933	WALL (OF HOUSE)	
+Nagano-2013-1256-241	241	0441	mud wall			
+Nagano-2013-1256-242	242	0443	window	1375	WINDOW	
+Nagano-2013-1256-243	243	0446	roof	769	ROOF	
+Nagano-2013-1256-244	244	0448	fireplace/fireplace (火塘)	185	FIREPLACE	
+Nagano-2013-1256-245	245	0450	fire	221	FIRE	
+Nagano-2013-1256-246	246	0452	light (vt), catch fire (v)	737	LIGHT (IGNITE)	
+Nagano-2013-1256-247	247	0454	flame	889	FLAME	
+Nagano-2013-1256-248	248	0456	smoke	778	SMOKE (EXHAUST)	
+Nagano-2013-1256-249	249	0458	soot	830	SOOT	
+Nagano-2013-1256-250	250	0460	ashes	646	ASH	
+Nagano-2013-1256-251	251	0461	cinder(s)			Should be added to Concepticon.
+Nagano-2013-1256-252	252	0463	charcoal	1188	CHARCOAL	
+Nagano-2013-1256-253	253	0464	coal	2658	COAL	
+Nagano-2013-1256-254	254	0466	firewood	10	FIREWOOD	
+Nagano-2013-1256-255	255	0468	matches	1133	MATCH	
+Nagano-2013-1256-256	256	0469	matches	1133	MATCH	
+Nagano-2013-1256-257	257	0471	extinguish (vt)	127	EXTINGUISH	
+Nagano-2013-1256-258	258	0472	put out (vt)			
+Nagano-2013-1256-259	259	0473	go out, fade (vi)	2704	FADE (LOOSE COLOR)	
+Nagano-2013-1256-260	260	0475	burn (vi)	1428	BURNING	
+Nagano-2013-1256-261	261	0476	get burned (vi)			
+Nagano-2013-1256-262	262	0478	set fire to (vt)			
```

Comment:
2258 SET ALIGHT?

Reply:
Yes, you are right.

Reply:
Which is deprecated in favor of 737	LIGHT (IGNITE)